frontend/model/getters.js

```diff
@@ -230,16 +241,19 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }) =>
       return payments.sort((paymentA, paymentB) => paymentB.height - paymentA.height)
     }
   },
-  ourPaymentsReceivedInPeriod (state, getters) {
-    return (period) => {
+  ourPaymentsSentInPeriod (state, getters) {
+    return (period) => getters.ourPaymentsSentInPeriodForGroup(getters.currentGroupState, period)
+  },
+  ourPaymentsReceivedInPeriodForGroup (state, getters) {
+    return (state, period) => {
```

Comment:
Right below the unchanged line referencing `getters.groupPeriodPayments` uses current group state